modulefiles/gaeac6.lua

```diff
@@ -0,0 +1,68 @@
+help([[
+loads TC_tracker modulefile and related set environment veriables on GAEA C6
+]])
+
+prepend_path("MODULEPATH", "/autofs/ncrc-svm1_proj/epic/spack-stack/spack-stack-1.6.0/envs/unified-env-c6/install/modulefiles/Core")
+
+stack_intel_ver=os.getenv("stack_intel_ver") or "2023.2.0"
+load(pathJoin("stack-intel", stack_intel_ver))
+
+stack_mpich_ver=os.getenv("stack_mpich_ver") or "8.1.29"
+load(pathJoin("stack-cray-mpich", stack_mpich_ver))
+
+craype_ver=os.getenv("craype_ver") or "2.7.30"
+load(pathJoin("craype", craype_ver))
```

Comment:
I suggest in a later PR that you keep a list of versions in a version file since they are (generally) common to spack-stack versions